src/main/java/TaskManager.java

```diff
@@ -1,5 +1,39 @@
 public class TaskManager {
-    private static Task[] tasks = new Task[100];
+
+    private static final int MAX_TASK = 100;
+    private static final int TASK_NAME_INDEX = 0;
+    private static final int TASK_DATE_INDEX = 1;
+    private static final int TODO_NAME_START_INDEX = 5;
+    private static final int DEADLINE_NAME_START_INDEX = 9;
+    private static final int EVENT_NAME_START_INDEX = 6;
+    private static final int SPACES_BETWEEN_SLASH_AND_DATE = 4;
+
+    private static final String LS = System.lineSeparator();
```

Comment:
Good use of constants to make the code neater!

Reply:
I agree, the constant names are also clear and concise!